build/yamls/flow-visibility/flow-visibility.yaml

```diff
@@ -0,0 +1,399 @@
+          sum(reverseThroughput) AS reverseThroughput,
+          sum(throughputFromSourceNode) AS throughputFromSourceNode,
+          sum(reverseThroughputFromSourceNode) AS reverseThroughputFromSourceNode, 
+          sum(throughputFromDestinationNode) AS throughputFromDestinationNode, 
+          sum(reverseThroughputFromDestinationNode) AS reverseThroughputFromDestinationNode
+      FROM flows
+      GROUP BY
+          flowEndSeconds,
+          flowEndSecondsFromSourceNode,
+          flowEndSecondsFromDestinationNode,
+          sourceNodeName,
+          destinationNodeName,
+          sourcePodNamespace;
+
+      CREATE MATERIALIZED VIEW flows_policy_view
+      ENGINE = SummingMergeTree
+      ORDER BY (
+          flowEndSeconds,
+          flowEndSecondsFromSourceNode,
+          flowEndSecondsFromDestinationNode,
+          egressNetworkPolicyName,
+          egressNetworkPolicyRuleAction,
+          ingressNetworkPolicyName,
+          ingressNetworkPolicyRuleAction,
+          sourcePodNamespace)
+      POPULATE 
+      AS SELECT
+          flowEndSeconds,
+          flowEndSecondsFromSourceNode,
+          flowEndSecondsFromDestinationNode,
+          egressNetworkPolicyName,
+          egressNetworkPolicyRuleAction,
+          ingressNetworkPolicyName,
+          ingressNetworkPolicyRuleAction,
+          sourcePodNamespace,
+          sum(octetDeltaCount) AS octetDeltaCount,
+          sum(reverseOctetDeltaCount) AS reverseOctetDeltaCount,
+          sum(throughput) AS throughput,
+          sum(reverseThroughput) AS reverseThroughput,
+          sum(throughputFromSourceNode) AS throughputFromSourceNode,
+          sum(reverseThroughputFromSourceNode) AS reverseThroughputFromSourceNode, 
+          sum(throughputFromDestinationNode) AS throughputFromDestinationNode, 
+          sum(reverseThroughputFromDestinationNode) AS reverseThroughputFromDestinationNode
+      FROM flows
+      GROUP BY
+          flowEndSeconds,
+          flowEndSecondsFromSourceNode,
+          flowEndSecondsFromDestinationNode,
+          egressNetworkPolicyName,
+          egressNetworkPolicyRuleAction,
+          ingressNetworkPolicyName,
+          ingressNetworkPolicyRuleAction,
+          sourcePodNamespace;
+
+      CREATE TABLE IF NOT EXISTS recommendations (
+          id UUID,
+          type String,
+          timeCreated DateTime,
+          yamls String
+      ) engine=MergeTree
+      ORDER BY (timeCreated);
+    EOSQL
+---
+apiVersion: "clickhouse.altinity.com/v1"
+kind: "ClickHouseInstallation"
+metadata:
+  name: clickhouse
+  namespace: flow-visibility
+spec:
+  configuration:
+    clusters:
+      - name: "clickhouse"
+        layout:
+          shardsCount: 1
+          replicasCount: 1
+  defaults:
+    templates:
+      podTemplate: pod-template
+  templates:
+    podTemplates:
+      - name: pod-template
+        spec:
+          containers:
+            - name: clickhouse
+              image: yandex/clickhouse-server:20.7
```

Comment:
Sure, or maybe we could have this change in your storage configuration PR? Since this PR focus on the data schema and they may release new version before our storage configuration finalized.

Reply:
OK, would include that.